CHANGES.md

```diff
@@ -7,6 +7,7 @@ Release Notes.
 #### Project
 * Incompatible with previous releases when use H2/MySQL/TiDB storage options, due to support multiple alarm rules triggered for one entity.
 * Chore: adapt `create_source_release.sh` to make it runnable on Linux.
+* Add `package` to `.proto` files, prevent polluting top-level namespace in some languages; The OAP server still supports previous agents, whereas the previous OAP server (<=8.3.0) won't recognize newer agents since this version (>= 8.4.0).
```

Comment:
```suggestion
* Add `package` to `.proto` files, prevent polluting top-level namespace in some languages; The OAP server supports previous agent releases, whereas the previous OAP server (<=8.3.0) won't recognize newer agents since this version (>= 8.4.0).
```